File: JavaScript/penjumlahan-dua-bilangan.js
const prompt = require("prompt-sync")({ sigint: true });
let num1 = parseInt(prompt("Masukkan bilangan pertama: "));
let num2 = parseInt(prompt("Masukkan bilangan kedua: "));
let jumlah = num1 + num2;
console.log(num1 + " + " + num2 + " = " + jumlah);

File: C++/penjumlahan-dua-bilangan.cpp
#include <iostream>

using namespace std;

int main()
{
    int num1, num2, jumlah;

    cout << "Masukkan bilangan pertama: ";
    cin >> num1;
    cout << "Masukkan bilangan kedua: ";
    cin >> num2;

    jumlah = num1 + num2;

    cout << num1 << " + " << num2 << " = " << jumlah << endl;

    return 0;
}